### DISCUSSION TITLE
Modulo HTML

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#194] [Q&A]
Compañeros el apartado de html sigue sin desbloquearse completo?

**Comment by JuanSalvador73** [comment]
Solo "Elementos Compuestos" tiene información publicada al momento de escribir esto, las 21:19 hora CDMX. Puedes ir avanzando en la práctica de la pastelería mientras.

**Comment by doguedogue** [comment]
Pues si está completo, solo hay partes que están en: 

- html
- excel 
- video 

sigue los links

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/02%20-%20HTML

Ejemplo:
![image](https://user-images.githubusercontent.com/23409026/156487078-6cffe2ab-054c-4db4-91ef-a97faa75c7d0.png)
